test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/PrefixedAttributeTagHelpers_Runtime.codegen.cs

```diff
@@ -181,10 +181,6 @@ public class TestFiles_IntegrationTests_CodeGenerationIntegrationTest_PrefixedAt
             __tagHelperExecutionContext.AddTagHelperAttribute(__tagHelperAttribute_5);
             __TestNamespace_InputTagHelper2.StringDictionaryProperty["paprika"] = (string)__tagHelperAttribute_5.Value;
             __tagHelperExecutionContext.AddTagHelperAttribute(__tagHelperAttribute_5);
-            if (__TestNamespace_InputTagHelper1.StringDictionaryProperty == null)
```

Comment:
This is the only change to codegen, it removed a redundant check on this dictionary